release_notes.md

```diff
@@ -6,5 +6,6 @@
 
 ## Breaking changes
 
-## Dependency Changes 
+## Dependency Changes
 
+Removed dependency on Mono.Posix.NETStandard by instead relying on P/Invoke to generate inotify signals in Linux; reducing the size of the package (https://github.com/Azure/azure-functions-durable-extension/pull/1643)
```

Comment:
smallest of nits: I think you can just put #1643 instead of the hard link. You only need a hard link if you are referring to something from a separate repo like DurableTask framework.

Reply:
Will update, thanks!